core/base/document.php

```diff
@@ -1,17 +1,17 @@
 <?php
 namespace Elementor\Core\Base;
 
+use Elementor\Controls_Manager;
+use Elementor\Controls_Stack;
 use Elementor\Core\Files\CSS\Post as Post_CSS;
+use Elementor\Core\Settings\Manager as SettingsManager;
+use Elementor\Core\Settings\Page\Manager as PageManager;
 use Elementor\Core\Utils\Exceptions;
-use Elementor\Plugin;
 use Elementor\DB;
-use Elementor\Controls_Manager;
-use Elementor\Controls_Stack;
+use Elementor\Plugin;
```

Comment:
Cancel, leave it for a "Code Formatting" session.

Reply:
Handled.